docs/content/technical-specifications.md

```diff
@@ -85,6 +85,15 @@ We also provide NGINX Plus images through the AWS Marketplace. Please see [Using
 |Debian-based image with NGINX App Protect WAF and DoS | ``debian:11-slim`` | NGINX App Protect WAF and DoS, NGINX Plus JavaScript and OpenTracing modules, OpenTracing tracers for Jaeger, Zipkin and Datadog | [F5 NGINX Ingress Controller with F5 NGINX App Protect DoS](https://aws.amazon.com/marketplace/pp/prodview-sghjw2csktega) | amd64 |
 {{% /table %}}
 
+We also provide NGINX Plus images through the Google Cloud Marketplace. Please see [Using GCP Marketplace Ingress Controller](/nginx-ingress-controller/installation/using-gcp-marketplace-package/) for details on how use them.
```

Comment:
```suggestion
We also provide NGINX Plus images through the Google Cloud Marketplace. Please see [Using GCP Marketplace Ingress Controller](https://docs.nginx.com/nginx-ingress-controller/installation/using-gcp-marketplace-package/) for details on how use them.
```

Reply:
@shaun-nx I don't think a fully qualified ref is needed in this case as the ref is internal, eg: https://github.com/nginxinc/kubernetes-ingress/blob/v3.1.1/docs/content/installation/installation-with-manifests.md?plain=1#L19

Reply:
@vepatel Ah I see, thanks for noticing that!